### PR TITLE
Add AWS troubleshooting page and add into applicable pages

### PR DESCRIPTION
**Pull request by stevenGravy**
Added some common troubleshooting issues encountered with AWS connections.

### CODE REVIEW

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+### Certificate Error
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:1]:
```suggestion
### Certificate error
```
We use sentence case in subheadings

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+```
+
+AWS provides instructions to rotate your [SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html). Even if
+your database is using the `rda-ca-2019` certificate and was created before July 28, 2020 that 
+will require rotation.
+
+### No Credential Providers Error
+
+If you see the error ` NoCredentialProviders: no valid providers in chain` in logs then Teleport
+is not detecting the required Credentials to connect via AWS IAM permissions.  Check
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:16]:
```suggestion
is not detecting the required credentials to connect via AWS IAM permissions. Check whether
```

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+the credentials or security role has been applied in the machine running the Teleport database service.
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:17]:
```suggestion
the credentials or security role has been applied in the machine running the Teleport Database Service.
```

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+### Timeout Errors
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:19]:
```suggestion
### Timeout errors
```

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+The Teleport database service needs connectivity to the database endpoints.  That may require
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:21]:
```suggestion
The Teleport Database Service needs connectivity to your database endpoints. That may require
```

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+enabling inbound traffic within the VPC or routing rules. Using the `nc`
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:22]: The VPC where your database servers are running?

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+program you can verify connections to databases.
```
Comment (ptgott) [inline docs/pages/includes/database-access/aws-troubleshooting.mdx:23]:
```suggestion
program you can verify connections to databases:
```

File: docs/pages/includes/database-access/aws-troubleshooting.mdx (hunk 6 above)
Comment (stevenGravy) [reply docs/pages/includes/database-access/aws-troubleshooting.mdx]: @ptgott modified language to make more clear

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 8]
```diff
@@ -0,0 +1,28 @@
+### Certificate Error
+
+If your `tsh db connect` error includes the following text you likely have a RDS database created before July 28, 2020.
+
+```code
+x509: certificate relies on legacy Common Name field, use SANs instead
+```
+
+AWS provides instructions to rotate your [SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html). Even if
+your database is using the `rda-ca-2019` certificate and was created before July 28, 2020 that
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/pages/includes/database-access/aws-troubleshooting.mdx]: removed. AWS makes this clear

File: docs/pages/includes/database-access/aws-troubleshooting.mdx [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+x509: certificate relies on legacy Common Name field, use SANs instead
+```
+
+AWS provides instructions to rotate your [SSL/TLS certificate](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/UsingWithRDS.SSL-certificate-rotation.html). Even if
+your database is using the `rda-ca-2019` certificate and was created before July 28, 2020 that 
+will require rotation.
+
+### No Credential Providers Error
+
+If you see the error ` NoCredentialProviders: no valid providers in chain` in logs then Teleport
```
[the remark this answers is not in the document]
Comment (stevenGravy) [reply docs/pages/includes/database-access/aws-troubleshooting.mdx]: Database Service, updated @ptgott

**Review by ptgott — APPROVED** [review]

**Review by strideynet — APPROVED** [review]

**Pull request merged**